src/assets/css/problemsolved.css

```diff
@@ -0,0 +1,6 @@
+.progress-container {
+  margin-bottom: 100px;
+}
+h4 {
+  margin-bottom: auto;
+}
```

Comment:
Better wrap it in some parent div to avoid affecting global styling.

Reply:
Solved